src/Dao/PlagiabotDao.php

```diff
@@ -68,6 +69,11 @@ public function getPlagiarismRecords( $n = 50, $options ) {
 					$filters[] = "status IS NULL";
 					break;
 			}
+			// search filters
+			if ( $searchCriteria == 'page' && $searchText ) {
+				$searchText = mysql_escape_string( $searchText );
+				$filters[] = "page_title LIKE '%" . $searchText . "%'";
```

Comment:
I would give this a try again, as prepared statements are much safer:
```php
$filters[] = "page_title LIKE CONCAT('%', :searchtext, '%')";
$preparedParams['searchtext'] = str_replace(' ', '_', $searchText);
```
This worked for me on my local.

The only other problem I noticed was after clicking on "Load more", the search text is no longer applied.